src/System.Security.Cryptography.Cng/tests/RsaCngTests.cs

```diff
@@ -0,0 +1,283 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class RsaCngTests
+    {
+        [Fact]
+        public static void EncryptDecryptRoundtrip()
```

Comment:
The RSA algorithm tests should just source-include the versions in Common\test\Cryptography\AlgorithmImplementations\RSA; and you'll need to define a tiny bit of boilerplate to let the tests know how to build an RSACng.  https://github.com/dotnet/corefx/commit/dc473a18dcc13b845c08ceda2c98b9ac4854e064 is where I did it for OpenSSL.